ui/app/AppLayouts/Wallet/panels/BuyCryptoProvidersListPanel.qml

```diff
@@ -32,6 +32,7 @@ ColumnLayout {
 
     StatusSwitchTabBar {
         id: tabBar
+        visible: false
```

Comment:
Cool, just put a reference to this issue here, to explain why we're (temporarily?) hiding it

Reply:
added explanation